### DISCUSSION TITLE
Latin manuscript transcription in MediaWiki

### DISCUSSION THREAD

**Discussion by redheadkelly** [psb1558/Junicode-font#274] [Q&A] [answered]
I'm trying to mimic the transcription of a Latin manuscript that was printed in an old book. The characters used in the printing are identical to the ones used in this document that used to be found in Junicode's /docs directory.

[https://rosssurnameproject.net/charts/Sample_Statutes_transcript.pdf](https://rosssurnameproject.net/charts/Sample_Statutes_transcript.pdf)

I've created a Latin Junicode class in the wiki's MediaWiki:Common.css:

`.junicodelatin {
 	font-family:Junicode Two Beta VF, sans serif;
	font-weight: normal; /* that is, 400 */    
	font-feature-settings: "cv81" 2, "calt", "ccmp", "dlig", "liga", "loca", "rlig", "kern", "mark", "mkmk", "ss08";
}`

I've managed to create the curly zigzag with `t&#x034F;&#x035B;`.

But I'm not getting the appropriate tilde. I'm using `r&#x0303;` or `r&#x034F;&#x0303;`, but it's looking more like a macron than a tilde.

Is it possible for someone to just give me the list of the font-feature-settings that were used to create the PDF that I linked to above. That's exactly the set I need.

Thanks.

**Comment by kenmcd** [comment]
The tilde was changed at some point during the beta to be more curvy.
And it looks like you are using a very old beta font. 
`font-family:Junicode Two Beta VF`
Get the latest release (v2.206) from here: 
https://github.com/psb1558/Junicode-font/releases

This is what that font looks like now:
![tilde-examples](https://github.com/psb1558/Junicode-font/assets/844920/6a4c78b0-fb92-483a-8b37-de9e42285dba)

**Comment by psb1558** [comment, marked as answer]
The revisions to the combining tilde (U+0303) that @kenmcd mentions were made precisely because the tilde was losing its curves at small sizes (mainly, I think, in a document pointed out to me on WikiMedia). The shape was revised, and I added manual hinting to that glyph to prevent the straightening problem even at *super* low sizes. No CSS is needed here: just a recent version of Junicode (no longer "Junicode Two Beta" because it's no longer a beta).

Sorry you'll have to change "Junicode Two Beta" wherever it occurs in your CSS to "Junicode."

**Reply by redheadkelly** [reply]
Great. Thanks everyone. I did actually download the latest files the other day, but the ttf files appeared to be the same ones I already had installed on my laptop so I didn’t update them. I’m developing offline so those are the fonts being displayed for now. For the public site I’m dependent on the MediaWiki language extension that serves the font, which I believe is up-to-date. 

Is there something I need to do other than just download the latest  ttf file?

**Reply by redheadkelly** [reply]
Oh wait. I found it. The new versions were buried deep in a different directory than last time. 

Thanks again!

**Comment by psb1558** [comment]
There's nothing you need to do for the combining tilde: it should just work. But if any more questions come up, just launch a new thread right here.

**Reply by redheadkelly** [reply]
Well, I upgraded my font and the tilde looks great but I’ve lost my curly zigzag.

**Reply by psb1558** [reply]
Can you show what you're doing and what you want? The OpenType layout changed several times while the font was in beta (but it's stable now). Curly zigzag is now on cv81[2].

**Reply by redheadkelly** [reply]
Thank you . I appreciate your help.

This is the document I'm copying. No. 31 here.

[https://archive.org/details/liberinsulemissa85inchuoft/page/xlvii/mode/1up](https://archive.org/details/liberinsulemissa85inchuoft/page/xlvii/mode/1up)

This is the page I created.

[https://www.rosssurnameproject.net/wiki/index.php?title=Source:Scottish_document_Liber_Insule_Missarum,_no._31](https://www.rosssurnameproject.net/wiki/index.php?title=Source:Scottish_document_Liber_Insule_Missarum,_no._31)

My CSS is still
`.junicodelatin {
 	font-family:Junicode, sans serif;
	font-weight: normal; /* that is, 400 */    
	font-feature-settings: "cv81" 2, "calt", "ccmp", "dlig", "liga", "loca", "rlig", "kern", "mark", "mkmk", "ss08";
}`

I'm trying to create the curly zigzag. I've tried `t&#x035B;`, like you have above, and `t&#0304;&#xe0030;&#xe0035;`. I tried removing everything except `"cv81" 2` from the font-feature-settings.

I had it working before I updated the ttf on my laptop, so I'm not sure if there's something else I need to do.

Thanks again.

**Comment by psb1558** [comment]
If I understand what I'm seeing in your page, the version of Junicode being loaded is 1.002 (the shape of the tilde seems to confirm this). You need 2.206.

**Reply by redheadkelly** [reply]
Oh geez. You are so right. The public site that I sent you is having to retrieve the font through the MediaWiki language extension. Apparently, they are not up-to-date after all. I also just yesterday installed this extension on my local machine that is not connected to the internet and that was forcing my development site to also use that version. Once I disabled it, everything worked fine again.

Thanks for your patience. I appreciate it.
